otterdog/eclipse-velocitas.jsonnet

```diff
@@ -95,6 +106,10 @@ orgs.newOrg('eclipse-velocitas') {
     },
     orgs.newRepo('devenv-github-integration') {
       allow_update_branch: false,
+      archived: true,
```

Comment:
Did you check with @doosuu if this shall be archived?

Reply:
this is currently completely empty and we never used it...

If we need it we can bring it back from the archive

Reply:
Ok, if it's easy, let's do it.